Templates/DefaultProject/Template/Code/gem.json

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
Why are you removing the default projects gem.json?

Reply:
Because **you** moved it to the Gem folder: https://github.com/o3de/o3de/blob/development/Templates/DefaultProject/Template/Gem/gem.json

Reply:
got it.